### ISSUE TITLE
Gaps in boxes drown with double frames

### ISSUE THREAD

**Issue by VladimirMarkelov** [cormullion/juliamono#157]
Thank you for the great font! I love it. The only small issue is that a few (not all) double frame characters are drawn with gaps. 

Windows 10 Home, Font version 0.048 (I installed only 4 fonts: regular, bold, italic, and bold+italic):

![image](https://user-images.githubusercontent.com/1577186/219536719-3670b840-e743-4d9e-9d9e-2975c74a2e84.png)

I spotted the issue in Conemu with FAR running inside it. FAR uses double frames in the main window and the gaps at the bottom of the screen are always visible. I took an example from https://en.wikipedia.org/wiki/Box-drawing_character#Examples:

```
┌─┬┐  ╔═╦╗  ╓─╥╖  ╒═╤╕
│ ││  ║ ║║  ║ ║║  │ ││
├─┼┤  ╠═╬╣  ╟─╫╢  ╞═╪╡
└─┴┘  ╚═╩╝  ╙─╨╜  ╘═╧╛
┌───────────────────┐
│  ╔═══╗ Some Text  │▒
│  ╚═╦═╝ in the box │▒
╞═╤══╩══╤═══════════╡▒
│ ├──┬──┤           │▒
│ └──┴──┘           │▒
└───────────────────┘▒
 ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒
```

In tried few more applications(including `Notepad++` and `gVim`). The boxes above are displayed with gaps everywhere. On another laptop (different laptop brand and Windows version), the gaps are visible, too. I do not have MacOS at hand, so I cannot check it there. Changing the size of the font did not fix anything (moreover, on small sizes around 5-6, the gaps appears even for single frames, but I doubt 5-6 font size is usable, so it is not a problem at all).

It is a screenshot from Notepad++. As you see, only some bottom box parts have gaps. Top and middle ones are fine:

![image](https://user-images.githubusercontent.com/1577186/219537449-7d1e9475-c18f-485e-b9a9-5211b6af5763.png)

I found an old closed issue https://github.com/cormullion/juliamono/issues/109 . The issue suggested to remove ASCII versions of the font. In my case, I do not have any of them(see the first screenshot). I started using your font since version `0.047` and then updated to `0.048`. Both times I installed the same 4 font files.

If the issue is not reproducible at your machine, no problem - I can live with gaps 😄 

Thank you!

**Comment by cormullion** [comment]
Hi, Vladimir, thanks for the report! 

I could be wrong, but I'm guessing that you have the line spacing/leading set to 1.2 or more.

As far as I know most fonts won't draw boxes that connect vertically once the line spacing gets above a certain value, 1.2 here. (I've not found one yet.)

IBM Plex Mono:

<img width="686" alt="boxes-plex" src="https://user-images.githubusercontent.com/52289/219600516-f42e1392-b34b-4692-8fa2-acb267c0418e.png">

Cascadia Code:

<img width="564" alt="boes-cascadea" src="https://user-images.githubusercontent.com/52289/219600525-4d173618-5c67-41cf-bdee-c0bc8803834e.png">

Fira Code:

<img width="564" alt="boxes-firacode" src="https://user-images.githubusercontent.com/52289/219600532-e6e8259c-3b5e-4978-8e5d-3ab02bf4f967.png">

Some terminal applications (wezterm, kitty, perhaps) provide you with an option for drawing box characters directly, rather than relying on a font's box character drawing set. These will be drawn correctly, because the terminal knows exactly how widely separated the lines are, even if the font doesn't.

Having said that - if these other fonts don't show gaps on your system  but JuliaMono does, then I confess I'm puzzled as to the cause, and solution... 🤔

**Comment by VladimirMarkelov** [comment]
Yes, some fonts display the boxes correctly in Notepad++. Cascadia code looks similar to what your posted. But Fira Code looks good:

![image](https://user-images.githubusercontent.com/1577186/219617135-dff9f950-2aa8-42c5-92fe-f00f3816605d.png)

I tried other fonts, and found fonts that shows correctly: Courier New, Dejavu Sans Mono, Liberation Mono, Victor Mono and a few more. Here is DejaVu Sans Mono:

![image](https://user-images.githubusercontent.com/1577186/219618004-20f622b7-4ee3-4e0d-af80-412f5a047126.png)

and Victor Mono:

![image](https://user-images.githubusercontent.com/1577186/219617877-92c45f90-8501-4a76-98d0-1a420b5fc6ff.png)

Some other fonts failed to show correct boxes: e.g, "Iosevka Fixed", "Input". "Hack"(though, Hack has different trouble).

I have no clue why it happens, too 😄  Thank for trying to help. It is a tiny trouble: do not waste much time on it. Feel free to close the issue.

**Comment by cormullion** [comment]
Thanks for the research. It’s puzzling. I wonder whether font hinting is contributing to the confusion… When I’m in “font design mode” these little inconsistencies bother me. I’ll leave this open for now, in case any new information comes up.

**Comment by Bolpat** [comment]
Some of the bold-light boxes are still too short on the top.

<img width="781" height="834" alt="Image" src="https://github.com/user-attachments/assets/58b9d630-52c0-4a5d-b079-dfba3292babc" />

These are the offending ones:
<pre>
┞ ┟  ┡ ┢
┦ ┧  ┩ ┪
┵ ┶  ┹ ┺
┽ ┾  ╇ ╈  ╉ ╊
</pre>
Their codes are
251E, 251F, 2521, 2522
2526, 2527, 2529, 252A
2535, 2536, 2539, 253A
253D, 253E, 2547, 2548, 2549, 254A
